cmd/gaiad/testnet.go

```diff
@@ -156,24 +157,13 @@ func InitTestnet(cmd *cobra.Command, config *tmconfig.Config, cdc *codec.Codec,
 		memo := fmt.Sprintf("%s@%s:26656", nodeIDs[i], ip)
 		genFiles = append(genFiles, config.GenesisFile())
 
-		buf := bufio.NewReader(cmd.InOrStdin())
-		prompt := fmt.Sprintf(
-			"Password for account '%s' (default %s):", nodeDirName, client.DefaultKeyPass,
-		)
-
-		keyPass, err := client.GetPassword(prompt, buf)
```

Comment:
ditto, is it not required anymore?

Reply:
Nope, thanks to `COSMOS_SDK_TEST_KEYRING = y` that creates a temporary passwordless keyring